utils/unleash.py

```diff
@@ -39,8 +43,11 @@ def get_feature_toggle_state(integration_name, defer=None):
     defer(lambda: client.destroy())
 
     # get feature toggle state
+    fallback_function = get_feature_toggle_default
```

Comment:
this could go in the else to avoid setting `fallback_function` twice.

Reply:
done!